### PR TITLE
My Portfolio: Boostrap ver

### PR DESCRIPTION
**Pull request by DanielAD10**
PR Description

Project Name: My Portfolio: Boostrap ver
Tech:Built with HTML & Boostrap

In this PR,

Added intex.html
Installed Boostrap
Styled with Boostrap

Thank you for your reviewing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,24 +8,26 @@
...
+                <button class="navbar-toggler justify-content-end" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo01" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+                    <ul class="navbar-nav nav nav-pills me-auto mb-2 mb-md-0">
+                        <li class="nav-item"><a class="nav-link" href="#scrollspyHeading1">Portfolio</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#scrollspyHeading2">About</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#scrollspyHeading3">Contact</a></li>
+                    </ul>
+                </div>
```
Comment (sagieramos) [inline index.html:23]: ### Consider incorporating semantic tags into your HTML markup like the ```section``` tage.
- Semantic tags provide meaningful structure and context to the content of a web page, allowing search engines and assistive technologies to better understand the purpose and hierarchy of various elements. By utilizing semantic tags, you can improve both the user experience and search engine optimization (SEO) of your website.

File: text.txt [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <h1>Multi-Post Stories Gain+Glory</h1>
+                        <ul>
```
Comment (akinmolaomotlani) [inline text.txt:63]: Please use the proper semantic tags. I realized you used h1 for a subheading, you can use other tags instead for a subheading. so you can have a well-structured code. follow the instruction.

File: text.txt [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
```
Comment (akinmolaomotlani) [inline text.txt:92]: please use the proper semantic tags.

File: text.txt [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+                </div>
+            </div>
+        </section>
+        <section>
+            <div>
+                <div>
+                    <h2>About me</h2>
+                    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                    <button>Get my resume</button>
```
Comment (akinmolaomotlani) [inline text.txt:128]: please let us know the type of button and the tag. so that the website can understand the function of the button

**Review by akinmolaomotlani — COMMENTED** [review]
Required changes♻️ : kindly make sure you make the proper changes on code.

**Review by akinmolaomotlani — COMMENTED** [review]
Required changes♻️ : kindly make sure you make the proper changes on code.

**Review by sagieramos — CHANGES_REQUESTED** [review]
### Required changes :recycle: 
 
Please check my comment

Thanks :grin:

**Pull request merged**